creators/views.py

```diff
@@ -111,3 +111,22 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message" : "KEYERROR"}, status=401)
 
+class ProductListView(View):
+    @login_decorator
+    def post(self,request):
+        user = request.user
+
+        user_id = Creator.objects.get(user=user.pk)
+        products = Product.objects.filter(creator=user_id).prefetch_related('productthumbnailmedia_set')
+
+        products = [{
+            'product_id' : product.pk,
+            'title'      : product.title,
+            'status'     : product.status.status,
+            'category'   : product.subcategory.name,
+            'created_at' : product.created_at,
+            'image_url'  : [product_image.media.storage_path for product_image in ProductThumbnailMedia.objects.filter(product_id=product.pk) ]
```

Comment:
두번째로 line 120에서 prefetch_related('productthumbnailmedia_set')을 하셨기 때문에 ` ProductThumbnailMedia.objects.filter(product_id=product.pk)`로 작성하게되면 의미가 없습니다!

그렇기 때문에 product instance에서부터 시작해서 접근해서 `productthumbnailmedia`를 가져오도록 해주셔야 합니다~
아래 코드를 참고해주세요!

```suggestion
            'image_url'  : [product_thumnail.media.storage_path for product_thumbnail in product.productthumbnailmedia_set.all()]
```